public/css/layout.css

```diff
@@ -0,0 +1,24 @@
+body {
+    background: #000;
+    margin: 0;
+}
+
+#screen {
+    display: block;
+    image-rendering: pixelated;
+    margin: auto;
+    width: 100vw;
+    max-height: 100vh;
+    --aspect-4-3: calc(3/4 * 100);
+    --aspect-16-9: calc(9/16 * 100);
```

Comment:
We multiply with 100 here so that we can write `1vw` on line #17. 

I would like us to remove this multiplier and use `100vw` on the classes remove the indirection.

Reply:
Obsolete. New commits added.